test/e2e/util.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func init() {
+	rand.Seed(time.Now().UTC().UnixNano())
+}
+
+// randomSuffix provides a random string to append to certain base name.
+func randomSuffix() string {
+	source := []rune("abcdefghijklmnopqrstuvwxyz")
```

Comment:
same comments from before re: random suffix unless I missed something else

Reply:
Before: Seed with current time in each `randomSuffix` function call. 2 parallel tests may end up using the same seed.
Now: Seed with current time in the `init` function. It should be fine now. Correct me if I'm wrong :)

Reply:
Fixed